var/spack/repos/builtin/packages/metall/package.py

```diff
@@ -16,14 +16,8 @@ class Metall(CMakePackage):
     version('master', branch='master')
     version('develop', branch='develop')
 
+    version('0.10', sha256='58b4b5507d4db5baca315b1bed2b728981755d755b91ef63bd0b6dfaf320f46b')
     version('0.9', sha256='2d7bd9ea2f1e04136050f210884445a9e3dcb96c992cf42ff9ea4b392f85f927')
-    version('0.8', sha256='02d7379a499ecd6ff7a3aaae436a2cd4dbe21c2fc23f5dad7449e7e8c832c77b')
```

Comment:
Is there a reason for removing the older releases from the package?

We tend to recommend they be kept in the package if and until they are no longer available.  This allows, for example, anyone who may want to build older versions to do so "safely" since the sha256 is retained.

Reply:
There are significant changes recently in Metall (including some big bug fixes). I want to remove the old versions so that users don't use them accidentally.